examples/app-pg/Dockerfile

```diff
@@ -5,4 +5,4 @@ WORKDIR /app
 # Copy code to container
 COPY . .
 
-CMD ["python", "main.py"]
+CMD ["python", "main.py", "3"]
```

Comment:
?

Reply:
It looks like he's passing the number of fake data to generate as an argument from here.

@yankeexe Let's not do that in build time. Instead, we can take the count from the environment during runtime and have a default count as a fallback.

Reply:
Also, the issue states a data volume of ~10,000 records. Let's set that as default maybe?

cc: @kabirbaidhya

Reply:
yes we need a multiplier or say an arg that gives the no. of rows that we can set to 10000 or larger.

Reply:
but that should be in the code and not here

Reply:
no need environment var for this, it could be a constant value

Reply:
Yes. We could take that from environment variables (`os.environ`) instead.

Reply:
A constant would be enough too.